### PR TITLE
Toolbar refactoring

### PR DESCRIPTION
**Pull request by IgorPopach**

### CODE REVIEW

**Review by Blager89 — APPROVED** [review]

File: client/src/components/Rooms/Rooms.scss [hunk 1]
```diff
@@ -1,9 +1,10 @@
 .rooms-page {
-    background-color: $black;
+    background-color: $white;
+    text-align: center;
 
     &-cards {
-        width: 90%;
-        margin: 0 auto;
+        width: 90% !important;
+        margin: 0 auto !important;
```
Comment (yodeco) [inline client/src/components/Rooms/Rooms.scss]: Why is important used here? #root ...selector helps set desired width?

File: client/src/components/Rooms/RoomCard/RoomCard.js [hunk 2]
```diff
@@ -1,31 +1,36 @@
...
-          <div>{props.category}</div>
-          <div>
-            {props.members} / {props.limit}
+    <Grid item xs={12} sm={6} md={4} className="col" ontouchstart="this.classList.toggle('hover');">
+      <div className="room-card" onClick={props.clicked}>
+        <div
+          className="front"
+          style={backgroundImage}
+        >
+          <img src={props.avatar} alt={props.avatar}/>
```
Comment (yodeco) [inline client/src/components/Rooms/RoomCard/RoomCard.js]: Why avatar link in alt attribute? Does this give any benefit?

File: client/src/components/Rooms/Rooms.js [hunk 3]
```diff
@@ -23,79 +23,98 @@ class Rooms extends Component {
...
     this.setState({ [event.target.name]: event.target.value });
   };
 
+  getDate = () => {
+    const currentDate = new Date();
+    const date = this.formatDate(currentDate);
+    this.setState({ date });
+  };
+
+  formatDate = d => {
```
Comment (yodeco) [inline client/src/components/Rooms/Rooms.js]: I beleive we should use moment.js for handling dates. Maybe it is more reasonable add it and convert date via moment.js?

**Review by yodeco — APPROVED** [review]

File: client/src/components/Rooms/RoomCard/RoomCard.js (hunk 2 above)
Comment (IgorPopach) [reply client/src/components/Rooms/RoomCard/RoomCard.js]: No, it doesn't...I use {props.avatar} like unique props for alt...we don't use it, it just for fixing alarm in console: "img elements must have an alt prop...."

File: client/src/components/Rooms/Rooms.scss (hunk 1 above)
Comment (IgorPopach) [reply client/src/components/Rooms/Rooms.scss]: fixed it...

**Pull request merged**